app/templates/search/_categories.html

```diff
@@ -1,14 +1,47 @@
-<ul>
-  {% for category in categories %}
-    <li>
-      {% if category.link %}<a href="{{ category.link }}">{% endif %}{% if category.selected %}<strong>{% endif %}
-        {{ category.label }}
-      {% if category.selected %}</strong>{% endif %}{% if category.link %}</a>{% endif %}
-      {% if category.children and category.selected %}
-        {% with categories=category.children %}
-          {% include 'search/_categories.html' %}
-        {% endwith %}
+{%- macro top_level_link(text, url) -%}
+  <a href="{{ url }}" class="lot-filters__top-level-link"><span class="visually-hidden">Back to </span>{{ text }}</a>
```

Comment:
Just to document our discussion, I suggested it might actually be clear enough from the context that a given category (or lot) name is more broad than a currently-selected sub-category. So the impact on the user's search results is clear enough in a screen-reading context, even without any introductory text like this at all .